docs/how-to/command-line-quick-start.md

```diff
@@ -176,89 +169,78 @@ Check out some of the other files in this directory:
+```bash
+ipfs cat /ipfs/QmW2WQi7j6c7UgJTarActp7tDNikE4B2qXtFCfLPdsgaTQ/cat.jpg > cat.jpg
 open cat.jpg
 ```
 
-Next try sending objects to the network, and then viewing it in your favorite browser. The example below uses `curl` as the browser, but you can open the IPFS URL in other browsers as well:
+Next, try sending objects to the network, and then viewing it in your favorite browser. The example below uses `curl` as the browser, but you can open the IPFS URL in other browsers as well:
 
-```sh
-> hash=`echo "I <3 IPFS -$(whoami)" | ipfs add -q`
-> curl "https://ipfs.io/ipfs/$hash"
-I <3 IPFS -<your username>
+```bash
+hash=`echo "I <3 IPFS -$(whoami)" | ipfs add -q`
+curl "https://ipfs.io/ipfs/$hash"
+> I <3 IPFS -<your username>
 ```
 
-Cool, huh? The gateway served a file _from your computer_. The gateway queried
-the Distributed hash table (DHT), found your machine, requested the file, your machine sent it to the
-gateway, and the gateway sent it to your browser.
+Cool, huh? The gateway served a file _from your computer_. The gateway queried the Distributed hash table (DHT), found your machine, requested the file, your computer sent it to the gateway, and the gateway sent it to your browser.
 
-<div class="alert alert-warning">
-    Depending on the state of the network, <code>curl</code> may take a while. The public gateways may be overloaded or having a hard time reaching you.
-</div>
+Depending on the state of the network, `curl` may take a while. The public gateways may be overloaded or having a hard time reaching you.
 
 You can also check it out at your own local gateway:
 
-```sh
-> curl "http://127.0.0.1:8080/ipfs/$hash"
-I <3 IPFS -<your username>
+```bash
+curl "http://127.0.0.1:8080/ipfs/$hash"
+> I <3 IPFS -<your username>
 ```
 
-By default, your gateway is not exposed to the world, it only works locally.
+By default, your gateway is not exposed to the world. It only works locally.
 
 ## Web console
 
-We also have a web console you can use to check the state of your node. In your favorite web browser, open:
-
-<pre><code><a href="http://localhost:5001/webui">http://localhost:5001/webui</a></code></pre>
-
-This should bring up a console like this:
+You can view the web console on your local node by going to [`localhost:5001/webui`](http://localhost:5001/webui). This should bring up a console like this:
 
 ![Web console connection view](https://docs.ipfs.io/introduction/assets/webui-connection.png)
 
 ## IPFS Companion
 
 While we are at it, [IPFS Companion](https://github.com/ipfs-shipyard/ipfs-companion#ipfs-companion) is a browser extension that simplifies access to IPFS resources and adds support for the IPFS protocol.
 
-It will automatically redirect IPFS gateway requests to your local daemon so that you are not relying on, or trusting, remote gateways.
+It will automatically redirect IPFS gateway requests to your local daemon so that you are not relying on or trusting, remote gateways.
```

Comment:
I think this comma needs to stay ...

Reply:
I'll just answer both comma issues here.

These checks popped up in Grammarly's default North America grammar rules. For the sake of consistency across multiple writers it makes sense to follow these rules.

Saying that, it would be great if we could define our own set of rules that Circle CI or VuePress could run when building the site. I've used spell-checkers from places like NPM before, and they work well. But I've never encountered a grammar-checker than can run against a git repo.

Reply:
OK.

Reply:
OK.

Reply:
OK.

Reply:
[Issue posted on `ipfs/docs`](https://github.com/ipfs/docs/issues/438).